prow/plugins/hold/hold.go

```diff
@@ -40,7 +40,7 @@ const (
 
 var (
 	labelRe       = regexp.MustCompile(`(?mi)^/hold(\s.*)?$`)
-	labelCancelRe = regexp.MustCompile(`(?mi)^/(hold\s+cancel|unhold)\s*$`)
+	labelCancelRe = regexp.MustCompile(`(?mi)^/(remove-hold)|(hold\s+cancel|unhold)\s*$`)
```

Comment:
The paretheses are inconsistent here, the three commands are equal so we should have `(one|two|three)`, not `(one)|(two|three)`